Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
```

Comment:
surely this won't have go installed?

Reply:
Doesn't need to - the build happens outside of the container, and just the statically compiled binary goes in